nerpa.py

```diff
@@ -284,7 +284,8 @@ def run(args, log):
     log.start()
 
     if args.predictions is not None:
-        path_predictions = copy_prediction_list(args, output_dir)
+        raise NotImplemented('Precomputed BGC predictions not supported yet')
```

Comment:
Did we ever (and do we have any plan to) support precomputed BGC predictions?

Reply:
Yes, we did, and probably it makes sense to have. If a user wishes to rerun the analysis of a huge number of antiSMASH predictions vs the new set of structures, s/he can just reuse already generated BGC variants. Now it should be super easy with just YAML dump/load functions.